openapi-validator/src/main/java/io/ballerina/openapi/validator/TypeSymbolToJsonValidatorUtil.java

```diff
@@ -194,14 +194,15 @@
                         TypeMismatch validationError = new TypeMismatch(fieldSymbol.getValue().getName().orElseThrow(),
                                 convertTypeToEnum(entry.getValue().getType()),
                                 convertTypeToEnum(fieldSymbol.getValue().typeDescriptor().typeKind().getName()),
-                                componentName, fieldSymbol.getValue().location());
+                                componentName, fieldSymbol.getValue().getLocation().get());
```

Comment:
Don't we need to check whether the location is empty? 

@pubudu91 In which contexts can we expect that the location being empty?

Reply:
Location would be empty for type symbols in general (except `TypeReferenceTypeSymbol`)